front/src/NotFound.tsx

```diff
@@ -0,0 +1,69 @@
+import { useNavigate } from 'react-router-dom';
+import styled from '@emotion/styled';
+
+import { MainButton } from '@/ui/button/components/MainButton';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+
+import { CompaniesMockMode } from './pages/companies/CompaniesMockMode';
+
+const BackDrop = styled.div`
```

Comment:
All styled components should be prefixed with Styled
so:
StyledBackDrop
StyledTextContainer...

This way we easily understand it's just a styled component while using it